### ISSUE TITLE
Iterating over trackers for a custom field raises exception

### ISSUE THREAD

**Issue by necrolyte2** [maxtepkeev/python-redmine#78]
```
>>> c = list(redmine.custom_field.all())
>>> c[0]
<redmine.resources.CustomField #34 "Encounter Date">
>>> c[0].trackers
<redmine.resultsets.ResourceSet object with Tracker resources>
>>> print list(c[0].trackers)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/tyghe/Projects/redmine_sample_management/env/lib/python2.7/site-packages/redmine/resultsets.py", line 107, in __len__
    return sum(1 for _ in self)
  File "/home/tyghe/Projects/redmine_sample_management/env/lib/python2.7/site-packages/redmine/resultsets.py", line 107, in <genexpr>
    return sum(1 for _ in self)
  File "/home/tyghe/Projects/redmine_sample_management/env/lib/python2.7/site-packages/redmine/resultsets.py", line 103, in <genexpr>
    return (self.manager.to_resource(resource) for resource in self.resources)
  File "/home/tyghe/Projects/redmine_sample_management/env/lib/python2.7/site-packages/redmine/managers.py", line 105, in to_resource
    return self.resource_class(self, resource)
  File "/home/tyghe/Projects/redmine_sample_management/env/lib/python2.7/site-packages/redmine/resources.py", line 125, in __init__
    self._attributes.update(attributes)
ValueError: dictionary update sequence element #0 has length 1; 2 is required
```

Here is the json from /custom_fields.json
```
{"custom_fields":[{"id":34,"name":"Encounter Date","customized_type":"issue","field_format":"date","regexp":"","visible":true,"trackers":{"tracker":{"id":19,"name":"Respiratory Questionaire"}},"roles":{}}
```

Redmine: 2.4.5-stable

**Comment by maxtepkeev** [comment]
Hi!

Works fine here:

```python
>>> list(redmine.custom_field.all()[6].trackers) 
[<redmine.resources.Tracker #4 "Bug">,
 <redmine.resources.Tracker #5 "Task">,
 <redmine.resources.Tracker #6 "Support">,
 <redmine.resources.Tracker #7 "User story">]
```

The JSON output from /custom_fields.json that you attached is incomplete, are you sure you've copy pasted it correctly ?

**Comment by necrolyte2** [comment]
I've only pasted the relevant part. I didn't include the ending 
```
]}
```

**Comment by maxtepkeev** [comment]
Ah, I see, I can confirm the error with your output. It's strange that my Redmine gives me different output to yours. I'll have to investigate this and returns to you with results ASAP.

**Comment by necrolyte2** [comment]
Sounds good. Seems like it is likely a redmine version difference then.
Just let me know what you need on this end to help

**Comment by maxtepkeev** [comment]
I believe I've found the answer.

It is a [bug](http://www.redmine.org/issues/16739) in Redmine that was fixed in 2.5.2, that is why our outputs differ (I'm on 2.6.0).

I can fix this on the python-redmine side so that it worked with Redmine versions <2.5.2 but I believe that you'll want to upgrade your installation if you need to get all trackers instead of only one.

**Comment by necrolyte2** [comment]
Ahh yes, ok

I'll have to upgrade then. Probably time to do that anyways

**Comment by maxtepkeev** [comment]
Fixed in 489b46a2df20a4aa88f7526c0f4528eba48cc5c1. Thanks for the report.

**Comment by necrolyte2** [comment]
Works great, thanks!
